test/e2e_tests/backend/ConversationRepository/conversationRepository.mocks.ts

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
helper to create conversations via API. This can be enhanced step by step and others can be added